docs/source/Support/bskReleaseNotes.rst

```diff
@@ -52,6 +52,7 @@ Version |release|
 - updated Eigen library to 3.4.0
 - updated OpenCV library to 4.5.5
 - Added documentation on using pre-commit formatters and clang formating
+- Added CI support to check that the pre-commit process passes
```

Comment:
It is my understanding that release notes communicate changes to the product. Changes to the CI are unrelated to the product. It seems redundant to add a release note for these kinds of changes.

Reply:
I see your point @patkenneally about keeping BSK release notes more use-facing.  

I do feel such changes to the commit process should be documented, but the release notes may not be the place.  How about adding a brief statement to the CONTRIBUTING.md file.  Here we discuss the pre-commit process that is expected if code is contributed to BSK.  I could state there that the CI process runs this as well to ensure this?  That way the developer shouldn't be surprised.

Reply:
I think that is a good idea. And thinking a little more, it makes sense because it really is part of the development process which the CONTRIBUTING.md speaks to.

Reply:
Ok, just deleted the release notes comment and pushed some text mentioning briefly that
pre-commit runs now on the CI servers with a PR request being made.